rpcs3/Emu/Cell/Modules/cellAudio.h

```diff
@@ -192,10 +192,10 @@ struct cell_audio_config
 	struct raw_config
 	{
 		bool buffering_enabled = false;
-		s64 desired_buffer_duration = 0;
```

Comment:
these variables were grouped by use case.
sorting them by type seems wrong

Reply:
Already fixed it, thank you

Reply:
Already fixed it, thank you